bld.bat

```diff
@@ -1,5 +1,9 @@
-mkdir %LIBRARY_PREFIX%\bin
-mkdir %LIBRARY_PREFIX%\lib
-copy %SRC_DIR%\bin\*.bat %LIBRARY_PREFIX%\bin
-copy %SRC_DIR%\lib\*.jar %LIBRARY_PREFIX%\lib
+set pkg=%PKG_NAME%-%PKG_VERSION%-%PKG_BUILDNUM%
+
+mkdir %PREFIX%\bin
```

Comment:
No idea whether the standard is `PREFIX` or `LIBRARY_PREFIX`, and/or whether it's the same or different on unix/windows.

Reply:
Pretty sure I had tried `PREFIX` first. But perhaps enough has changed here.